fixtures/completion/this_return_value.php

```diff
@@ -0,0 +1,20 @@
+<?php
+class Grand
+{
+  /** @return $this */
+  public function foo() {
+    return $this;
+  }
+}
+class Parent1 extends Grand {
+  
+}
+
+class Child extends Parent1 {
+  public function bar() {
+    $this->foo()->q
```

Comment:
could you format these consistently with the rest of the codebase? (4 spaces, brace on new line)